app/components/Header.tsx

```diff
@@ -46,12 +52,18 @@ const Header: FC = () => (
         </div>
 
         <div className="hidden md:block ml-5">
+          <NavLink
+            to={SIGNIN_LINK}
+            className="text-yellow-300 hover:text-indigo-50"
+          >
+            Sign in
+          </NavLink>
           <CallToAction
-            href={HOW_TO_LIST_PROJECT_URL}
-            rel="noopener"
-            target="_blank"
+            href={SIGNUP_LINK}
+            format={LINK_FORMATS.secondary}
+            inverse={true}
           >
-            List Your Project
+            Sign up
           </CallToAction>
         </div>
```

Comment:
**todo:** we don't have the ability to sign in yet, so could you please comment out this part of the nav?

Reply:
It was commented out.